pytest_dbfixtures/factories/postgresql.py

```diff
@@ -176,6 +176,11 @@ def postgresql_proc_fixture(request):
         init_postgresql_directory(
             postgresql_ctl, config.postgresql.user, datadir
         )
+
+
+        with open(os.path.join(datadir, 'pg_hba.conf'), 'a') as f:
```

Comment:
@brunsgaard separate commit for this change would be perfect and at the same time, could we limit that setting change to BSD systems only? There, if I'm correct, You still won't be able to connect to testing postgresql isntance outside of the machine, right?